src/sql/create_semantic_tables.sql

```diff
@@ -1,41 +1,52 @@
 create schema if not exists semantic;
```

Comment:
Could you remove the sql files from the *PR*? The sql files were already generated to reflect the org file.